tests/wpt/metadata/url/urlsearchparams-constructor.any.js.ini

```diff
@@ -1,51 +1,8 @@
 [urlsearchparams-constructor.any.worker.html]
-  [Construct with object with two keys]
-    expected: FAIL
-
-  [Construct with object with NULL, non-ASCII, and surrogate keys]
```

Comment:
I've observed the `Construct with object with two keys` case and `Construct with object with NULL, non-ASCII, and surrogate keys` case will be `PASS`ed intermittently in my local.

I belive the root cause would be related to the `MozMap` type is `HashMap` which is an unordered collection so we might get result in different order when using map 🤔

Reply:
That sounds like a really good idea!

Reply:
Yeah, let me file an issue for it! :)

Reply:
Filed at #22557